lib/rules/length-zero-no-unit/index.mjs

```diff
@@ -206,6 +194,13 @@ function isZero(number) {
 	return Number.parseFloat(number) === 0;
 }
 
+/**
+ * @param {string} unit
+ */
+function isNumber(unit) {
```

Comment:
This follows the convention in this file.

Reply:
[suggest] I think `isNumber` is confusing with `isNumber` in `validateTypes.mjs`. How about this?

```suggestion
function isNumberUnit(unit) {
```

Of course, I'm also okay with renaming to `isLengthUnit` and `isFractionUnit` for consistency.

Reply:
It's named `isNumber` in 2 other rules so I think it's fine as is.
i.e. `hue-degree-notation` and `lightness-notation`